metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -53,8 +54,9 @@ type kubernetesConfig struct {
 	KubeConfig        string                       `config:"kube_config"`
 	KubeClientOptions kubernetes.KubeClientOptions `config:"kube_client_options"`
 
-	Node       string        `config:"node"`
-	SyncPeriod time.Duration `config:"sync_period"`
+	Node                string        `config:"node"`
+	SyncPeriod          time.Duration `config:"sync_period"`
+	CacheExpirationTime time.Duration `config:"cache_expiration_time"`
```

Comment:
This would also need to be added and explained in the respective docs.

Reply:
https://github.com/elastic/beats/blob/d983d4a00aef40fd5fac4188dd4444a89da7b282/metricbeat/docs/modules/kubernetes.asciidoc